tests/acceptance/test_mode_transitions.py

```diff
@@ -43,13 +42,13 @@ def test_antenna_trans_to_stow(cm_manager_connected_to_antnn):
     """Test the stow function"""
     cm_manager_connected_to_antnn.set_stow_mode()
     # The antenna takes roughly 30 sec to switch modes
-    time.sleep(33)
+    time.sleep(45)
     assert (
         cm_manager_connected_to_antnn.get_antenna_mode() == Mode.STOW
     )
     assert (
         cm_manager_connected_to_antnn.get_antenna_func_state()
-        == FuncState.BRAKED
+        == FuncState.ESTOP
```

Comment:
When the set_stow_mode() is called, the functional state goes to Bracked. Has Pieter released a new simulator with new changes?

Reply:
It shifts between ESTOP and BRAKED, you can look at the logs of the tests that failed. You can also run it and check the GUI

Reply:
It's not meant to transition to ESTOP. Check the document that Pieter send for the mode transtions.

Reply:
Please double-check if an incorrect value is not sent into the simulator because the test has been stable for months.